modules/prediction/container/pose/pose_container.h

```diff
@@ -54,13 +54,31 @@ class PoseContainer : public Container {
    * @brief Transform pose to a perception obstacle.
    * @return A pointer to a perception obstacle.
    */
-  apollo::perception::PerceptionObstacle* ToPerceptionObstacle();
+  perception::PerceptionObstacle* ToPerceptionObstacle();
 
   /**
    * @brief Get timestamp
    */
   double GetTimestamp();
 
+  /**
+   * @brief Get speed of adc
```

Comment:
minor ,abbreviation could be made upper case.
i.e ADC for autonomous driving car.

Reply:
Thanks for the comments. It will be updated in next version.